### PR TITLE
Upcomingevents

### PR DESCRIPTION
**Pull request by joe-niu**

### CODE REVIEW

**Comment by mcjcloud** [comment]
It looks like you did not use the Angular CLI to create the upcoming events component.

Try deleting the "upcomingevents" folder you created.
Then, navigate to the "app" folder of the frontend and run:
`ng g c upcoming-events`
This will generate a new component for you and hook everything up properly.

Additionally, this ticket will require some backend changes in order to work properly but in the meantime just work on making the UI look like the mockups on Figma.

**Review by mcjcloud — CHANGES_REQUESTED** [review]
It looks like you did not use the Angular CLI to create the upcoming events component.

Try deleting the "upcomingevents" folder you created.
Then, navigate to the "app" folder of the frontend and run:
`ng g c upcoming-events`
This will generate a new component for you and hook everything up properly.

Additionally, this ticket will require some backend changes in order to work properly but in the meantime just work on making the UI look like the mockups on Figma.

File: ebay-frontend/src/app/upcomingevents/upcomingevents.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>Upcoming Events</p>
```
Comment (mcjcloud) [inline ebay-frontend/src/app/upcomingevents/upcomingevents.component.html:1]: The "upcomingevents" folder this is in should be deleted.

File: ebay-frontend/src/app/upcomingevents/upcomingevents.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    styleUrls: ['./upcomingevents.component.scss']
+  })
+  export class UpcomingEventsComponent implements OnInit {
+  
+    constructor() { }
+  
+    ngOnInit() {
+    }
+  
+  }
```
Comment (mcjcloud) [inline ebay-frontend/src/app/upcomingevents/upcomingevents.ts:15]: This file should be deleted

File: ebay-frontend/src/app/upcoming-events/upcoming-events.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>upcoming-events works!</p>
```
Comment (mcjcloud) [inline ebay-frontend/src/app/upcoming-events/upcoming-events.component.html:1]: This does not create the page found on Figma

**Review by mcjcloud — CHANGES_REQUESTED** [review]

**Comment by joe-niu** [comment]
WIP Pull Request

**Pull request closed without merging**